helm/cloudflare-tunnel-ingress-controller/templates/controlled-cloudflared-servicemonitor.yaml

```
@@ -0,0 +1,38 @@
+{{- if .Values.cloudflaredServiceMonitor.create }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "cloudflare-tunnel-ingress-controller.fullname" . }}-controlled-cloudflared
+  labels:
+    {{- include "cloudflare-tunnel-ingress-controller.labels" . | nindent 4 }}
+spec:
+  {{- if .Values.cloudflaredServiceMonitor.jobLabel }}
+  jobLabel: {{ .Values.cloudflaredServiceMonitor.jobLabel }}
+  {{- end }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: controlled-cloudflared
+      {{- include "cloudflare-tunnel-ingress-controller.selectorLabels" . | nindent 6 }}
+  endpoints:
+    - port: 44483
```

```suggestion
    - port: 'metrics'
```

when I test this PR on my local machine, it should use the name of port instead of use port directly

---

Quotes aren't needed here, so I'll add commit without them. Thanks for the suggestion.